### PR TITLE
Address  CVE-2019-0820

### PR DESCRIPTION
**Pull request by RussKie**
See https://github.com/advisories/GHSA-cmhx-cq75-c4mj

image

## Merge strategy

I agree that the maintainer squash merge this PR (if the commit message is clear).

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

File: Directory.Packages.props [hunk 1]
```diff
@@ -30,25 +30,26 @@
     <PackageVersion Include="System.Reactive.Linq" Version="5.0.0" />
     <PackageVersion Include="System.Reactive.Interfaces" Version="5.0.0" />
     <PackageVersion Include="System.ValueTuple" Version="4.5.0" /> <!-- Required by GitExtensions.Plugins.AutoCompileSubmodules -->
-
+
```
Comment (gerhardol) [inline Directory.Packages.props:33]: nit pick spaces

**Review by gerhardol — APPROVED** [review]

**Review by gerhardol — CHANGES_REQUESTED** [review]
build fails
https://ci.appveyor.com/project/gitextensions/gitextensions/builds/50955847#L245
Could not find reference "C:\Users\appveyor\.nuget\packages\netstandard.library\2.0.3\build\netstandard2.0.3\ref\System.Runtime.dll"

**Comment by mstv** [comment]
local build with VS 17.5.11 works for me without issues

**Comment by mstv** [comment]
`dotnet build` searches for
`"C:\Program Files\dotnet\sdk\NuGetFallbackFolder\netstandard.library\2.0.3\build\netstandard2.0.3\ref\System.Runtime.dll"`
but the existing folder is named
`"C:\Program Files\dotnet\sdk\NuGetFallbackFolder\netstandard.library\2.0.3\build\netstandard2.0\"`

**Comment by RussKie** [comment]
Thank you, folks.

**Review by mstv — APPROVED** [review]

**Review by gerhardol — APPROVED** [review]

**Pull request merged**